packages/ui/src/components/modals/ConditionModal.tsx

```diff
@@ -50,9 +49,15 @@ const convertConditionTypesToDropdownOptions = (conditionTypes: object): Operato
         })
 }
 
-const operatorOptions = convertConditionTypesToDropdownOptions(CLM.ConditionType)
+const allOperatorOptions = convertConditionTypesToDropdownOptions(CLM.ConditionType)
 // We know EQUAL will be found since it was created from enum type in line above
-const equalOperatorOption = operatorOptions.find(o => o.data === CLM.ConditionType.EQUAL)!
+const equalOperatorOption = allOperatorOptions.find(o => o.data === CLM.ConditionType.EQUAL)!
+const stringEqualOperatorOption = allOperatorOptions.find(o => o.data === CLM.ConditionType.STRING_EQUAL)!
+const stringEqualOperatorOptions = allOperatorOptions.filter(o => o.data === CLM.ConditionType.STRING_EQUAL)!
+const arithmeticOperatorOptions = allOperatorOptions.filter(o => o.data !== CLM.ConditionType.STRING_EQUAL)!
+var operatorOptions = allOperatorOptions
```

Comment:
`var`... 

It seems weird for this variable to not be state managed, but perhaps thats ordering issue.  Would check that.

Also, the `operatorOptions` likely isn't needed.
Based on the UI the user already constraints the operator types to three sets:

Entity Class | Comparison Type | Operators | Value
--- | --- | --- | ---
enum | string | == | "string"
multivalue | number | [==, !=, >, <, etc] | 234
custom, has number resolver | number | [==, !=, >, <, etc] | 98
.   | string | == | "my value"
(all other classes) | string | == | "my string"

I this case there is never a need to "change" the available operators to the DropDown control, you simply select it and make it disabled if it's constrained to equality (such as equals)

I thought the introduction of "comparison type" eliminated the need to have second equality operator.
![image](https://user-images.githubusercontent.com/2856501/74883922-e12e6c00-5326-11ea-937f-601bda21b609.png)

I would prefer that say "==" but not sure what you and @LarsLiden decided.

Also, when you select enum there seems to be bug with comparison type being undefined. Enums work based on "string" value equality so I thought it should say "string".  I'm not sure if this was intentional but think users might wonder. 

![image](https://user-images.githubusercontent.com/2856501/74883822-a0365780-5326-11ea-916d-5ac420836187.png)

Reply:
>I would prefer that say "==" but not sure what you and @LarsLiden decided.
We had "decided" (to the extent that we agreed on Teams, and no one else objected) to have "Matches exactly" be the operator for string types.

>Also, when you select enum there seems to be bug with comparison type being undefined

For this one, it's not undefined, it's simply not a choice (currently) for enum types.  Again that's what was "decided"  over Teams (same as above... to the extent that it was discussed with @LarsLiden who agreed, and no one objected).

Reply:
Hmm, I thought you had proposed "matches exactly" before he proposed the design which introduced new "comparison type" dropdown and now that the dropdown was added, the new operator no longer needed to differential the types of equals.
Anyways, can let him confirm what he wanted.

> For this one, it's not undefined, it's simply not a choice (currently) for enum types. Again that's what was "decided" over Teams (same as above... to the extent that it was discussed with @LarsLiden who agreed, and no one objected).

This seems contradicting.  Not having a choice because it's constrained to single choice is not the same as there being no choices.  Example, when someone select string comparison type, you constraint the operators to "matches exactly" and disable the dropdown, yet here you use the argument that if the user can't pick any options, don't show any options.  It does not look good for dropdowns to have no value. It makes the UI look broken. (I think we also have this issue on Action Mocking modal since the options get removed as entities are added until they run out)

When enum entity is selected, the comparison type is similarly forced to be something. Currently it's string comparison type using value of ENUM, but could have a separate enum type if you want that.

Reply:
It occurred to me, that our required conditions are ANDed which means if you have contradition conditions the action will never be valid.
Is there any consideration where a user can choose options to disregard casing? 
"Matches Exactly" will mean "abc != Abc" and given the variations that people type in the bot, I would imagine it's a common to normalize to upper or lower.  Or is this not required for PVA integration?

Reply:
> It does not look good for dropdowns to have no value. It makes the UI look broken.... 
 When enum entity is selected, the comparison type is similarly forced to be something. Currently it's string comparison type using value of ENUM, but could have a separate enum type if you want that.

I can change it so that for enums, it's string comparison type.

Reply:
Yes, that's what I was asking for. It makes more sense to me to put something there. I think Enum or Strings. but since enums are basically using string equivalence. I thought we could re-use that.

Reply:
done

Reply:
Ah here is he comment I was referring to.  It seems like you only made the comparison type string when enum entity is selected but didn't address the other parts of comment about changing list of options

Reply:
Also, on code reviews it's generally the responsibility of person who created comment to resolve.  GitHub's reviewing mechanism isn't as sophisticated as CodeFlow or ADO, so we don't have rules for github but it's kind of etiquette.

If it's simple and you're sure, then you can resolve but otherwise I'd ask.

Reply:
makes sense, sorry...